testing/cups-epson-escpr/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# $Id$
+# Maintainer: Michael Wyraz <michael@wyraz.de>
+# Adapated from https://aur.archlinux.org/packages/epson-inkjet-printer-escpr/
+#
+
+pkgname=cups-epson-escpr
+pkgver=1.6.29
+pkgrel=1
+pkgdesc="Epson Inkjet Printer Driver (ESC/P-R) for Linux"
+arch="all"
+url="http://download.ebz.epson.net/dsc/search/01/search/?OSC=LX"
+license="GPL2"
+depends="cups ghostscript"
+makedepends="cups-dev"
+options="!libtool"
+source="https://download3.ebz.epson.net/dsc/f/03/00/08/01/30/6cd80f9555b252d5d7101d450bb43a6a6e7f9689/epson-inkjet-printer-escpr-1.6.29-1lsb3.2.tar.gz build_x86_64.patch"
```

Comment:
It's customary to have additional source declarations on a new line, indented with a tab.